src/main/webapp/app/overview/course-overview.component.html

```diff
@@ -8,13 +8,29 @@
                         <div id="container" class="d-flex p-3 align-items-center text-decoration-none" [title]="course?.title">
                             <div ngbDropdown container="body" class="d-flex">
                                 @if (course && course.courseIcon) {
-                                    <div ngbDropdownToggle class="d-flex align-items-center justify-content-center pointer">
-                                        <jhi-secured-image [src]="course.courseIcon" />
-                                    </div>
+                                    @if (courses !== undefined) {
+                                        @if (courses.length > 0) {
```

Comment:
Why are we not displaying the course image anymore if courses is not defined?
I think that the check in line 11 should be added to the condition in line 12 (respectively line 23 and 24) - is there a problem with just writing it in line 12 as `@if (courses?.length > 0) {` ?

Reply:
your right i changed it 👍